client/src/main.rs

```diff
@@ -1793,6 +1844,40 @@ fn get_attestee_count(api: &Api, key: CommunityCeremony) -> ParticipantIndexType
 		.unwrap_or(0)
 }
 
+fn get_reputables_for_community_ceremony(
+	api: &Api,
+	community_ceremony: CommunityCeremony,
+	at_block: Option<Hash>,
+) -> Vec<AccountId> {
+	let key_prefix = api
+		.get_storage_double_map_key_prefix(
+			"EncointerCeremonies",
+			"ReputableRegistry",
+			community_ceremony,
+		)
+		.unwrap();
+	let max_keys = u32::MAX;
```

Comment:
changing this to 1000 solves the issue. but 1000 will not be enough for long

Reply:
or is this just keys per page?

Reply:
for now, I just log an error if the limit is reached. can fix that later using a nice wrapper fn

Reply:
...actually, it would be nice if the api-client would provide this wrapper fn as it generalizes well ;-)

Reply:
Good point, thanks !